### DISCUSSION TITLE
Kindly requesting some Noob guides

### DISCUSSION THREAD

**Discussion by nadimaj** [v2fly/fhs-install-v2ray#257] [General]
After installing v2ray via the script, no service is running or installed
What to check and what to do next?

**Comment by IceCodeNew** [comment]
https://github.com/v2fly/fhs-install-v2ray/blob/a9c7b8d4912b1fd184fdac908d3884972d8b0df7/install-release.sh#L630

You need to read the output of the script, not the guides.

**Reply by nadimaj** [reply]
Many thanks. Too much Windows lately
